### PR TITLE
Add sum function

### LINKED ISSUE #3: Add Python code for a sum function
It's nice all those planets, but have you considered adding some actual code?

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #3

### CODE REVIEW

File: arithmetic.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def sum(a, b):
```
Comment (sjvrijn) [inline arithmetic.py:1]: Shall we add a docstring to this function?

**Review by sjvrijn — CHANGES_REQUESTED** [review]

File: arithmetic.py (hunk 1 above)
Comment (svenvanderburg) [reply arithmetic.py:1]: Added in https://github.com/sjvrijn/planets/pull/4/commits/cc53c2d647d998cd79be579c65cfaafdb5dd2574

**Comment by svenvanderburg** [comment]
@sjvrijn I fixed your remarks can you merge it if you agree?

**Review by sjvrijn — APPROVED** [review]

**Comment by sjvrijn** [comment]
Thanks @svenvanderburg !

**Pull request merged**